install/kubernetes/agent/agent-tls-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-agent-tls
+data:
+  ca.crt: ""
+  tls.crt: ""
+  tls.key: ""
+type: Opaque
+
```

Comment:
I don't think this should be part of the installation manifests, since it contains eventually secret data (`tls.key`) and eventually is managed by an external entity such as ESO.